pandas/tests/series/test_datetime_values.py

```diff
@@ -548,3 +548,9 @@ def test_minmax_nat_series(self, nat):
     def test_minmax_nat_dataframe(self, nat):
         assert nat.min()[0] is pd.NaT
         assert nat.max()[0] is pd.NaT
+
+    def test_set_dt_with_string_index(self):
+        from datetime import date
+        x = pd.Series([1, 2, 3], index=['Date', 'b', 'other'])
+        x.Date = date.today()
```

Comment:
can you try this both as (parametrize)
``x.Date = `` and ``x['Date'] = ``

Reply:
I can't seem to figure out how to parametrize both these cases, as "Date" is being used in different contexts. What would be the variable(s) I parametrize and how would I use it with `x` to cover both cases?